pkg/runner/runner.go

```diff
@@ -72,17 +73,22 @@ func getNewHintRunnerContext(program *Program, userArgs []starknet.CairoFuncArgs
 	//  * segment_arena_ptr + 3
 	// But the builtin itself shouldn't be included in len(builtins), therefore the writeApOffset should be increased by 3.
```

Comment:
Can you check if this comment needs to be updated after the changes in the code?

Reply:
after the changes the comment is irrelevant. we strip the builtins from gas, segment arena and system, and hold that information in program